app/screens/receive-bitcoin-screen/receive-screen.tsx

```diff
@@ -0,0 +1,302 @@
+          disabled={!request.canSetReceivingWalletDescriptor}
+        />
+
+        <QRView
+          type={request.info?.data?.invoiceType || Invoice.OnChain}
+          getFullUri={request.info?.data?.getFullUriFn}
+          loading={request.state === PaymentRequestState.Loading}
+          completed={request.state === PaymentRequestState.Paid}
+          err={
+            request.state === PaymentRequestState.Error ? LL.ReceiveScreen.error() : ""
+          }
+          style={styles.qrView}
+          expired={request.state === PaymentRequestState.Expired}
+          regenerateInvoiceFn={request.regenerateInvoice}
+          copyToClipboard={request.copyToClipboard}
+          isPayCode={request.type === Invoice.PayCode}
+          canUsePayCode={request.canUsePaycode}
+          toggleIsSetLightningAddressModalVisible={
+            request.toggleIsSetLightningAddressModalVisible
+          }
+        />
+
+        <View style={styles.invoiceActions}>
+          {request.state !== PaymentRequestState.Loading &&
+            (request.type !== Invoice.PayCode ||
+              (request.type === Invoice.PayCode && request.canUsePaycode)) && (
+              <>
+                <View style={styles.copyInvoiceContainer}>
+                  <TouchableOpacity
+                    {...testProps(LL.ReceiveScreen.copyInvoice())}
+                    onPress={request.copyToClipboard}
+                  >
+                    <Text {...testProps("Copy Invoice")} color={colors.grey2}>
+                      <Icon color={colors.grey2} name="copy-outline" />
+                      <Text> </Text>
+                      {LL.ReceiveScreen.copyInvoice()}
+                    </Text>
+                  </TouchableOpacity>
+                </View>
+                <TouchableOpacity onPress={request.copyToClipboard}>
+                  <View>
+                    {request.readablePaymentRequest && (
+                      <Text
+                        {...testProps("readable-payment-request")}
+                        color={colors.grey2}
+                      >
+                        {request.readablePaymentRequest}
+                      </Text>
+                    )}
+                  </View>
+                </TouchableOpacity>
+                <View style={styles.shareInvoiceContainer}>
+                  <TouchableOpacity
+                    {...testProps(LL.ReceiveScreen.shareInvoice())}
+                    onPress={request.share}
+                  >
+                    <Text {...testProps("Share Invoice")} color={colors.grey2}>
+                      <Icon color={colors.grey2} name="share-outline" />
+                      <Text> </Text>
+                      {LL.ReceiveScreen.shareInvoice()}
+                    </Text>
+                  </TouchableOpacity>
+                </View>
+              </>
+            )}
+        </View>
+
+        <View style={styles.extraDetails}>
+          <Text>{request.extraDetails}</Text>
```

Comment:
When the timer starts running its width changes due to the changing numbers. Since the extra details are centered it causes them to wiggle around.

Reply:
How should we go about this problem?